src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
Creating a new instance of Random every time the `getRandomColor` method is called can be inefficient. Consider storing a single instance of Random as a private class field and reusing it.